### ISSUE TITLE
Streamers not found (Result of all fixes for me of error 5027)

### ISSUE THREAD

**Issue by Grimtsujigiri** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#617]
The first error I get is error 5027 upon trying to fix that through any of the recommended ways I get an issue where it goes through all of the streamers and says streamer not found. Even attempting to manually go into the cookie to find the auth token and adjust the code has not fixed the second issue. As soon as it has gone through the list of streamers and failed to find any of them the script then crashes with a bad auth message. 

To me, it seems like it's not properly able to use the auth token from the cookie or doesn't actually pull it correctly.

I have tried using this on two separate computers with the same issue.
 - OS: Windows 10
 - Python 3.10.7

Browsers attempted:
- Chrome
- Firefox

Edit:
The first image is it failing to download my follower list
The second image is with a manually set streamer and letting it run for a bit.
![image](https://user-images.githubusercontent.com/114784994/194224398-e0701a9b-99a9-4fcc-8e19-6a6f42b04cd2.png)
![image](https://user-images.githubusercontent.com/114784994/194224507-1e4bd403-87f6-413a-87bc-85c92c97fe6c.png)

**Comment by LukysGaming** [comment]
+1, twitch fucked astronomically something up

**Comment by tiilopes** [comment]
Try delete cookies?

**Comment by LukysGaming** [comment]
> Try delete cookies?
> or
> Check this https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/issues/606#issuecomment-1256839239

Chrome is fresh install

**Comment by Grimtsujigiri** [comment]
> Try delete cookies? or Check this [#606 (comment)](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/issues/606#issuecomment-1256839239)

Already tried both. Also tried the fix for dedicated servers where you manually go into the cookie and copy the auth token to the code

**Comment by tiilopes** [comment]
Try to enter a valid live stream name in example.py

**Comment by LukysGaming** [comment]
> Try to enter a valid live stream name in example.py

all stream names i have are valid

**Comment by Grimtsujigiri** [comment]
> Try to enter a valid live stream name in example.py

I have filled out the entire list, part of the list, and none of the list relying on the download of followers for the account all of that changes nothing.

**Comment by Grimtsujigiri** [comment]
The issue was caused by a weird incompatibility/cross-compatibility with a c++ compiler that running it with venv did not stop. Essentially what was happening was that the required libraries would be downloaded but for some reason, it would try to use older ones that came with the C++ compiler and were incompatible with the twitch miner. This also led to earlier problems with the build wheel section of installing requirements but I was able to circumvent that problem and had originally thought that it was past.

**Comment by VelesSage** [comment]
hi ,Grimtsujigiri how did you solve the problem, tell me in more detail, I also write

C:\Python\Twitch-Channel-Points-Miner-v2-master>python example1.py
11/10/22 14:54:13 - INFO - [run]: Start session: '07e89e3a-3f20-4d1f-9f66-20a4f243f31d'
11/10/22 14:54:13 - INFO - [run]: Loading data for 11 streamers. Please wait...
11/10/22 14:54:14 - INFO - [run]: Streamer chap does not exist
11/10/22 14:54:15 - INFO - [run]: Streamer streamer-username02 does not exist
11/10/22 14:54:16 - INFO - [run]: Streamer streamer-username03 does not exist
11/10/22 14:54:17 - INFO - [run]: Streamer streamer-username04 does not exist
11/10/22 14:54:18 - INFO - [run]: Streamer streamer-username05 does not exist
11/10/22 14:54:18 - INFO - [run]: Streamer streamer-username06 does not exist
11/10/22 14:54:19 - INFO - [run]: Streamer streamer-username07 does not exist
11/10/22 14:54:20 - INFO - [run]: Streamer streamer-username08 does not exist
11/10/22 14:54:21 - INFO - [run]: Streamer streamer-username09 does not exist
11/10/22 14:54:22 - INFO - [run]: Streamer streamer-username10 does not exist
11/10/22 14:54:23 - INFO - [run]: Streamer streamer-username11 does not exist
11/10/22 14:54:24 - WARNING - [warning]: websocket connected
11/10/22 14:54:24 - ERROR - [error]: error from callback <function WebSocketsPool.on_message at 0x000002520B80A1F0>: Error while trying to listen for a topic: {'type': 'RESPONSE', 'error': 'ERR_BADAUTH', 'nonce': 'B3vf8DV0fZH88GeSrtRFX9mEnsK3Ge'}
11/10/22 14:54:24 - ERROR - [on_error]: #0 - WebSocket error: Error while trying to listen for a topic: {'type': 'RESPONSE', 'error': 'ERR_BADAUTH', 'nonce': 'B3vf8DV0fZH88GeSrtRFX9mEnsK3Ge'}

**Comment by Grimtsujigiri** [comment]
I had a thing called msys2 installed and it had some library files that matched up with the required files. Basically whenever it tried to run, it was running on an old dependency.

To fix this I uninstalled msys2, cleared cache/cookies, and re-logged in on firefox which for me is a browser I almost never use. I also made sure that the two-factor authentication I have was the code from the app and not sms. (Don't know if that's specific to the fix but that's how it worked for me and I haven't touched it since.
